packages/js/e2e-utils-playwright/src/themes.js

```diff
@@ -0,0 +1,32 @@
+const { request } = require( '@playwright/test' );
+
+import { encodeCredentials } from './plugin-utils';
```

Comment:
Also relies on that extra plugin, so we should not include it in a general utilities package.